packages/block-library/src/index.js

```diff
@@ -87,6 +87,9 @@ import * as postFeaturedImage from './post-featured-image';
 import * as postHierarchicalTerms from './post-hierarchical-terms';
 import * as postTags from './post-tags';
 
+const isClassicEnabled =
+	window.wpEditorL10n && window.wp && window.wp.oldEditor;
+
```

Comment:
These values might change during runtime so a function would work better here. This would fix the tests too.
```suggestion
const isClassicEnabled = () =>
	window.wpEditorL10n && window.wp && window.wp.oldEditor;

```